packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -25,10 +25,11 @@ export interface TableProps {
     tableName?: string;
 
     /**
-     * The Type of Stream to create.
-     * @default ''
+     * When an item in the table is modified, StreamViewType determines what information
+     * is written to the stream for this table. Valid values for StreamViewType are:
+     * @default undefined
```

Comment:
Describe what the default _behavior_ is. In this case, "streams are disabled for this table" or something along these lines